### ISSUE TITLE
parse_version_tag can't take None (which is sent by read_version_tag) 

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/pepper#37]
```
Traceback (most recent call last):
  File "/home/arthur/.virtualenvs/pepper/bin/pepper", line 10, in <module>
    execfile(__file__)
  File "/home/arthur/local/salt/pepper/scripts/pepper", line 10, in <module>
    from pepper.cli import PepperCli
  File "/home/arthur/local/salt/pepper/pepper/__init__.py", line 26, in <module>
    version, sha = setup.get_version()
  File "/home/arthur/local/salt/pepper/pepper/../setup.py", line 90, in get_version
    version, num_commits, sha = parse_version_tag(read_version_tag())
  File "/home/arthur/local/salt/pepper/pepper/../setup.py", line 72, in parse_version_tag
    if '-g' not in tag:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by whiteinge** [comment]
Fixed in 2d3ae6d. Please verify the fix on the latest release.

**Comment by arthurlogilab** [comment]
Sorry, I seemed to have been stuck on version 0.2.0 for some git command problem. Now on proper head.
